pkgs/applications/networking/instant-messengers/trillian-im/default.nix

```diff
@@ -0,0 +1,70 @@
+{ lib
+, stdenv
+, fetchurl
+, pkgs
+}:
+
+let
+  version = "6.3.0.1";
+  src =
+    if stdenv.hostPlatform.system == "x86_64-linux" then
+      fetchurl {
+        urls = [
+          "https://www.trillian.im/get/linux/6.3/trillian_${version}_amd64.deb"
+        ];
+        sha256 = "42e3466ee236ac2644907059f0961eba3a6ed6b6156afb2c57f54ebe6065ac6f";
+      }
+    else
+      throw "Trillian for Linux is not supported on ${stdenv.hostPlatform.system}";
+
+in stdenv.mkDerivation {
```

Comment:
Don't be afraid of using `stdenv.mkDerivation rec` when adding things to Nixpkgs.